build-config/checkstyle/build.gradle

```
@@ -0,0 +1,68 @@
+///*
+// * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// *
+// * Licensed under the Apache License, Version 2.0 (the "License");
+// * you may not use this file except in compliance with the License.
+// * You may obtain a copy of the License at
+// *
+// * http://www.apache.org/licenses/LICENSE-2.0
+// *
+// * Unless required by applicable law or agreed to in writing, software
+// * distributed under the License is distributed on an "AS IS" BASIS,
+// * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// * See the License for the specific language governing permissions and
+// * limitations under the License.
+// *
+// */
+//
+//plugins {
+//    id "de.undercouch.download"
+//}
+//
+//apply plugin: 'java'
+//
+//task downloadMultipleFiles(type: Download) {
+//    src([
+//            'https://raw.githubusercontent.com/wso2/code-quality-tools/v1.4/checkstyle/jdk-17/checkstyle.xml',
+//            'https://raw.githubusercontent.com/wso2/code-quality-tools/v1.4/checkstyle/jdk-17/suppressions.xml'
+//    ])
+//    overwrite false
+//    onlyIfNewer true
+//    dest buildDir
+//}
+//
+//jar {
+//    enabled = false
+//}
+//
+//clean {
+//    enabled = false
+//}
+//
+//artifacts.add('default', file("${rootDir}/build-config/checkstyle/checkstyle.xml")) {
+//    builtBy('downloadMultipleFiles')
+//}
+//
+//artifacts.add('default', file("${rootDir}/build-config/checkstyle/suppressions.xml")) {
+//    builtBy('downloadMultipleFiles')
+//}
+
+plugins {
+    id 'java'
+}
+
+group = 'org.ballerinalang'
+version = '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation platform('org.junit:junit-bom:5.10.0')
+    testImplementation 'org.junit.jupiter:junit-jupiter'
+}
+
+test {
+    useJUnitPlatform()
+}
```

Lets add a newline at the EOF. Check for other places as well.
```suggestion
}

```